jgiven-spock2/src/main/groovy/com/tngtech/jgiven/spock/ScenarioSpec.groovy

```diff
@@ -0,0 +1,58 @@
+package com.tngtech.jgiven.spock
+
+import com.google.common.reflect.TypeToken
+import com.tngtech.jgiven.annotation.DoNotIntercept
+import com.tngtech.jgiven.impl.Scenario
+import com.tngtech.jgiven.junit.JGivenClassRule
+import com.tngtech.jgiven.spock.junit.JGivenSpockMethodRule
+import net.bytebuddy.ByteBuddy
+import net.bytebuddy.description.annotation.AnnotationDescription
+import net.bytebuddy.dynamic.loading.ClassLoadingStrategy
+import net.bytebuddy.implementation.SuperMethodCall
+import net.bytebuddy.matcher.ElementMatchers
+import org.junit.ClassRule
+import org.junit.Rule
+import spock.lang.Shared
+import spock.lang.Specification
+
+class ScenarioSpec<GIVEN, WHEN, THEN> extends Specification {
```

Comment:
Maybe add javadoc and an `@since` annotation here aswell? After all, the entire module will only be available from 1.2.0 onwards